packages/editor/src/components/page-attributes/parent.js

```diff
@@ -66,6 +66,7 @@ const applyWithSelect = withSelect( ( select ) => {
 		parent_exclude: postId,
 		orderby: 'menu_order',
 		order: 'asc',
+		_fields: 'id,title,parent',
```

Comment:
This is something we tried but it's not a valid approach because the data module keeps a cache per id and if you do this and another component request the same entity but want access to the remaining properties, it will get a trimmed version causing issues. Basically whenever you use `getEntityRecords` you shouldn't explicitly pass `context` or `fields`. This PR tried to solve it https://github.com/WordPress/gutenberg/pull/19498 but it's more complex than it seems.

Reply:
Well that's a bit of a bummer. Why not prevent caching for requests where `_fields` is used? And what should be done about `context` which is already being passed as `edit`.  Or should the request use a completely different approach - such as an AJAX request that calls wp_dropdown_pages, which is what this code is trying to emulate after all.

Reply:
I agree that it's not great. Ideally, something like #19498 is pursued to completion and support multiple entities shapes in core-data.

I also think the unlimited API requests is one of the worst aspects of Gutenberg right now and it has other solutions: pagination and I believe @epiqueras and @adamsilverstein are also looking into this issue.

Reply:
Given that the information returned for this `_fields` request is not going to be useful elsewhere, would it be possible to store it as a different entity type?

Reply:
To land something like #19498, we would need to reinvent GraphQL, as can be seen from the caveats that led to the closing of the PR.

Can we use a direct API call in this component until pagination lands?